src/js/components/search/table/ResultsTable.jsx

```diff
@@ -74,12 +75,17 @@ export default class ResultsTable extends React.Component {
             value: this.props.results[rowIndex][columnId],
             dataType: awardTableColumnTypes[columnId]
         };
-
         if (column.columnName === 'Award ID') {
             cellClass = ResultsTableLinkCell;
             props.id = this.props.results[rowIndex].generated_internal_id;
             props.column = 'award';
         }
+        else if ((column.columnName === 'Sub-Award ID') && this.props.subaward) {
```

Comment:
I want to check for subaward because that is the property we set to know if a user is on the sub award table. I do not really see Sub-Award ID making it to the award table, but I also think is important to clarify that we are on the subaward table. If you really want this removed, I will remove it.